Assets/Scripts/BoardSelect.cs

```diff
@@ -0,0 +1,198 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BoardSelect : MonoBehaviour
+{
+    [SerializeField] private GameObject[] _groundGameObject;
+    private int height; // tp the right
+    private int width; // to the left
+
+    private int[,] _floorplan =
+    {
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {1,1,1,3,1,4,4,1,1,1 },
+        {2,2,2,5,2,7,7,2,2,2 },
+        {1,1,1,3,1,4,4,1,1,1 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {1,1,1,3,1,4,4,1,1,1 },
+        {4,4,4,6,4,4,4,4,4,4 },
+        {4,4,4,6,4,4,4,4,4,4 },
+        {1,1,1,3,1,4,4,1,1,1 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+        {0,0,1,3,1,4,4,1,0,0 },
+    };
+
+
+    private GameObject[,] _floorGameObject;
+    private Ground[,] _floorGround;
+
+    private int _xSelect = 1;
+    private int _ySelect = 1;
+    private int _xPreviousSelect = 0;
+    private int _yPreviousSelect = 0;
+
+    private float _elapsedTime;
+    private float _timeStep = 0.02f;
+    private bool _isMoving = false;
+    private float _movingTime;
+    private float _movingDelay = 0.2f;
+
+private enum State { };
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        FillFloor();
+    }
+
+    // Update is called once per frame
+    void FixedUpdate()
```

Comment:
Is dit om het minder snel te laten bewegen?

Reply:
si